Parsely/wp-parsely
# Add "Quick Action Link" to the Dashboard in the Post List

## Description

Include a link to the statistics page for an article in the wp-admin Posts List. This uses the [Find in Parse.ly API Endpoint](https://www.parse.ly/help/integration/other-tools#find-in-parsely-api-endpoint).

Our filtering function is hooked into both the `post_row_actions` & the `page_row_actions` so it will do its thing on hierarchical and non lists.

## Motivation and Context



To make it easier to browse to the article page in the dashboard from within the WordPress interface.

## How Has This Been Tested?




* Browse to `/wp-admin/edit.php`
* There should be links to the Parse.ly Dashboard for each published post
* Browse to `/wp-admin/edit.php?post_type=page`
* There should be links to the Parse.ly Dashboard for each published page
* Browse to the wp-admin list for a "publicly-queryable" custom post type
* There should be links to the Parse.ly Dashboard for each published page
* Browse to the wp-admin list for a custom post type that is NOT "publicly-queryable"
* There should **NOT** be links to the Parse.ly Dashboard for each published page

## Screenshots (if appropriate):

|Link to `/find` endpoint|No Link on Private Posts|No Link on Drafts|
|---|---|---|
|![Screen Shot 2021-05-21 at 4 27 10 PM](https://user-images.githubusercontent.com/1587282/119198356-2f18af00-ba57-11eb-9ec2-80b35a962b95.png)|<img width="248" alt="Screen Shot 2021-05-24 at 2 22 42 PM" src="https://user-images.githubusercontent.com/1587282/119391121-f74e7900-bc9b-11eb-82c0-756b1209c7a6.png">|<img width="249" alt="Screen Shot 2021-05-24 at 2 22 49 PM" src="https://user-images.githubusercontent.com/1587282/119391145-fddcf080-bc9b-11eb-915b-409f951a4587.png">|




## Types of changes

New feature (non-breaking change which adds functionality)
